x-pack/platform/plugins/shared/triggers_actions_ui/public/application/hooks/use_load_rule_event_logs.ts

```diff
@@ -61,7 +61,7 @@ export function useLoadRuleEventLogs(props: UseLoadRuleEventLogsProps) {
     retry: 0,
     refetchOnWindowFocus: false,
   });
-  const hasExceedLogs = useMemo(() => error && error.body.statusCode === 413, [error]);
+  const hasExceedLogs = useMemo(() => error && error.body?.statusCode === 413, [error]);
```

Comment:
This is the main fix, the other two instances are just preemptive.

Q: What would happen if there is an error other than 413 in this case?